### DISCUSSION TITLE
Supabase and Cloudflare

### DISCUSSION THREAD

**Discussion by nida** [supabase/supabase#6062] [Questions]
We have been trying to use Supabase Auth for Github. Our domains are on Cloudflare. 

1. Auth for Google login works without any issue
2. In a local setup, direct IP access, everything works fine. 

The moment the login is accessed via domain we run into a CORS issue: 

POST /api/v1/ssoLogin HTTP/2
Host: [<subdomain>.<domain>.io](http://<subdomain>.<domain>.io/)
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:98.0) Gecko/20100101 Firefox/98.0
Accept: application/json, text/plain, */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br
Content-Type: application/json
Content-Length: 133
Origin: https://<host_subdomain>.<domain>.io
Connection: keep-alive
Referer: https://<host_subdomain>.<domain>.io/
Sec-Fetch-Dest: empty
Sec-Fetch-Mode: cors
Sec-Fetch-Site: same-site
TE: trailers

What happened?
There is an unknown connection issue between Cloudflare and the origin web server. As a result, the web page can not be displayed.
What can I do?
If you are a visitor of this website:
Please try again in a few minutes.
If you are the owner of this website:
There is an issue between Cloudflare's cache and your origin web server. Cloudflare monitors for these errors and automatically investigates the cause. To help support the investigation, you can pull the corresponding error log from your web server and submit it our support team. Please include the Ray ID (which is at the bottom of this error page). Additional troubleshooting resources.
Was this page helpful?
Thank you for your feedback!

Would any one have any recommendations or are missing out something? 

Thanks,
Nida

**Comment by Sulomus** [comment]
Hello! This is exactly what we are facing now - did you find any solution?
@nida
